data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/S3ScanPartitionCreationSupplier.java

```diff
@@ -116,10 +116,14 @@ private List<PartitionIdentifier> listFilteredS3ObjectsForBucket(final List<Stri
                     .map(objectKey -> PartitionIdentifier.builder().withPartitionKey(String.format(BUCKET_OBJECT_PARTITION_KEY_FORMAT, bucket, objectKey)).build())
                     .collect(Collectors.toList()));
 
+            LOG.info("Found page of {} objects from bucket {}", listObjectsV2Response.keyCount(), bucket);
+
             mostRecentLastModifiedTimestamp = getMostRecentLastModifiedTimestamp(listObjectsV2Response, mostRecentLastModifiedTimestamp);
         } while (listObjectsV2Response.isTruncated());
 
         globalStateMap.put(bucket, Objects.nonNull(mostRecentLastModifiedTimestamp) ? mostRecentLastModifiedTimestamp.toString() : null);
+
+        LOG.info("Returning partitions for {} s3 objects from bucket {}", allPartitionIdentifiers.size(), bucket);
```

Comment:
Minor: We should probably capitalize the "s" in "S3" as that is the name of the service. We can use "s3" when referring to the plugin itself.